cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -19,6 +19,7 @@ package kubeadm
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kubeletconfigv1alpha1 "k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig/v1alpha1"
+	"k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig"
```

Comment:
I'm sorry I didn't spot this right away, but this needs to be the external version
```go
kubeproxyconfigv1alpha1 "k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1"
```

that will fix the fact that you manually include `apiVersion` and  `kind` below (which you shouldn't)